internal/service/federatedsettingsidentityprovider/data_source_federated_settings_identity_provider_test.go

```diff
@@ -29,9 +28,9 @@ func TestAccFedDSFederatedSettingsIdentityProvider_samlBasic(t *testing.T) {
 					resource.TestCheckResourceAttrSet(resourceName, "associated_orgs.#"),
 					resource.TestCheckResourceAttrSet(resourceName, "acs_url"),
 					resource.TestCheckResourceAttrSet(resourceName, "display_name"),
```

Comment:
resource.TestCheckResourceAttrSet(resourceName, "display_name"),
is redundandt if you use TestCheckResourceAttr later to check the value

Reply:
Done